jdbc/src/main/java/nextstep/jdbc/JdbcTemplate.java

```diff
@@ -14,4 +19,61 @@ public class JdbcTemplate {
     public JdbcTemplate(final DataSource dataSource) {
         this.dataSource = dataSource;
     }
+
+    public void update(final String sql, final Object... parameters) {
+        log.debug("query : {}", sql);
+        execute(sql, ps -> {
+            setPreparedStatement(ps, parameters);
+            return ps.executeUpdate();
+        });
+    }
+
+    public <T> List<T> query(final String sql, final RowMapper<T> rowMapper, final Object... parameters) {
+        log.debug("query : {}", sql);
+        return execute(sql, ps -> {
+            setPreparedStatement(ps, parameters);
+            ResultSet rs = ps.executeQuery();
+            return extractData(rowMapper, rs);
+        });
+    }
+
+    public <T> T queryForObject(final String sql, final RowMapper<T> rowMapper, final Object... parameters) {
+        List<T> results = query(sql, rowMapper, parameters);
+        return singleResult(results);
```

Comment:
```suggestion
        return singleResult(query(sql, rowMapper, parameters));
```
반영 안하셔도 되지만 저는 요렇게 한 군데에서만 쓰이면 주로 인라인으로 적는게 깔끔해 보이더라구요!